src/Resources/scaffolds/6.0/user.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'description' => 'Create a basic user and unit test.',
+    'packages' => [
+        'orm' => 'all',
```

Comment:
The problem with using Flex aliases here is that these are ALWAYS installed. What I mean is: the `isPackageInstalled()` method always returns false for this. So if you, for example, ran `make:scaffold user` and then `make:scaffold register`, the latter will (of course) re-run the `user` scaffold... but that `user` scaffold is slow because it tries to reinstall `orm`, `security` and `phpunit`.